site/src/scaffold/styles/pagination.scss

```diff
@@ -0,0 +1,218 @@
+$font-size-jumbo: $font-size-0 !default;
```

Comment:
Heads up, we can't copy legacy components into scaffold. If they're referenced in our documentation / code examples, they need to be supported. Two options (TL;DR: migrate this, or delete):
1. Migrate this component full and update it to follow the size and component naming guidelines.
2. Or remove all references to it from examples in the case, and figure out another way to do the active class naming

Reply:
removed